### ISSUE TITLE
Fix setting codes

### ISSUE THREAD

**Issue by KOLANICH** [Lukasa/hyper#332]
```h2\.settings\.([A-Z_]{4,})``` -> ```h2.settings.SettingCodes.$1```

**Comment by Lukasa** [comment]
Yup, seems like we'd need this changed. Want to open a PR to do it?

**Comment by KOLANICH** [comment]
Sorry, I have some tech problems and can't do that now. Feel free to do it for me.
